cylc/uiserver/data_store_mgr.py

```diff
@@ -94,10 +94,12 @@ def update_contact(self, w_id, contact_data=None):
             flow.status = SuiteStatus.STOPPED.value
 
         # Apply to existing workflow data
-        apply_delta(WORKFLOW, delta, self.data[w_id])
         if 'delta_times' not in self.data[w_id]:
-            self.data[w_id]['delta_times'] = {}
-        self.data[w_id]['delta_times'][WORKFLOW] = delta.time
+            self.data[w_id]['delta_times'] = {
+                key: 0.0
+                for key in DATA_TEMPLATE
+            }
```

Comment:
Set all delta times to `0.0`, unless there is a delta time in `self.data` in which case do nothing?

Reply:
Yeah might only need WORKFLOW key created.

Reply:
Done.